xarray/core/variable.py

```diff
@@ -412,15 +473,15 @@ def __setitem__(self, key, value):
 
         See __getitem__ for more details.
         """
-        key = self._item_key_to_tuple(key)
+        dims, index_tuple = self._broadcast_indexes(key)
         if isinstance(self._data, dask_array_type):
             raise TypeError("this variable's data is stored in a dask array, "
```

Comment:
@shoyer 
Currently, `array[0, 1] = 1` gives `NotImplementedError: Item assignment with <class 'tuple'> not supported`

I couldn't find its API. [This one](http://dask.pydata.org/en/latest/array-api.html#dask.array.store)?